source/blog/_posts/2018-07-25-switching-to-composer-installation.md

```diff
@@ -0,0 +1,93 @@
+---
+title: An easy example how to switch from classic installation to composer installation
+tags:
+- Deployment
+- Composer Installation
+
+categories: 
+- dev
+
+authors: [tn]
+github_link: /blog/_posts/2018-07-25-switching-to-composer-installation.md
+
+---
+
+In this blog post you will learn how to change your Shopware classic installation to the composer installation. I already mentioned it in the shopware [Livestream](https://www.youtube.com/watch?v=oUME-FnlUKE) (a new format you should know ;-) ).
```

Comment:
I'd get rid of the "new format" thing as "new" will be outdated in some weeks ;)